libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -1435,3 +1461,5 @@ method initPubSub*(g: GossipSub) =
   g.lastFanoutPubSub = initTable[string, Moment]()  # last publish time for fanout topics
   g.gossip = initTable[string, seq[ControlIHave]]() # pending gossip
   g.control = initTable[string, ControlMessage]()   # pending control messages
+  var rng = newRng()
+  g.randomBytes = Curve25519Key.random(rng[]).getBytes()
```

Comment:
just use the RNG directly? no need to go via curve25519...

Reply:
forgot about it, that was supposed to be temporary indeed!